deploy/templates/identity/kdc.yaml

```diff
@@ -48,7 +48,7 @@ spec:
         # Do as little work as possible as root... We need to chown the storage
         # as the default permissions will not be correct.
         - name: chown-storage
-          image: "{{ .Values.identity.identity.image.registry }}/{{ .Values.identity.identity.image.repository }}:{{ .Values.identity.identity.image.tag }}"
+          image: "{{ include "amrc-connectivity-stack.image-name" .Values.identity.identity }}"
```

Comment:
Is there a reason these aren't using `amrc-connectivity-stack.image`, which includes `imagePullPolicy` from the values file?

Reply:
No. I needed to use the `image-name` one to include the suffix for the manager and it slipped through with them. I'll update now.